tests/bsim/bluetooth/host/adv/periodic/tests_scripts/per_adv_app_not_scanning.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Basic periodic advertising sync test: an advertiser advertises with periodic
+# advertising, and a scanner scans for and syncs to the periodic advertising.
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+simulation_id="per_adv"
+verbosity_level=2
+EXECUTE_TIMEOUT=10
```

Comment:
Please don't set EXECUTE_TIMEOUT if it is anyhow very small. It will default to 30. (same for the test below)